src/client/Microsoft.Identity.Client/Extensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Extension methods
+    /// </summary>
+    public static class Extensions // TODO MSAL5: move these to their parent types
```

Comment:
What about `OsCapabilitiesExtensions`?
I know that when using the IntelliSense, the class won't appear. However when looking at the reference documentation, customers will see the extension class name, and it's better if it's meaninful.

Reply:
+1

---
In reply to: [589412204](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2458#discussion_r589412204) [](ancestors = 589412204)